scripts/get-version

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I think, this script is simple enough to move it into the Makefile (it is currently used only there, anyway).
This would obsolete the cwd check, since the context of the Makefile is known.